nativeauthenticator/nativeauthenticator.py

```diff
@@ -1,18 +1,25 @@
 import bcrypt
-from jupyterhub.orm import User
 from jupyterhub.auth import Authenticator
 
 from sqlalchemy import inspect
-from sqlalchemy.orm import relationship
 from tornado import gen
+from traitlets import Bool
 
+from .common_credentials import COMMON_CREDENTIALS
```

Comment:
Can you make this a .txt file instead of a .py file?

Reply:
I thought about this but I would have to everytime open a file, create a list and then search in the list. I thought that if I already had a list, it would be more efficient. What do you think?

Reply:
Not sure how to do this using cache. Could you give me an example?

Reply:
Sure! something like

`common_passwords.py`

```
COMMON_PASSWORDS = None

def is_password_common(password):
    global COMMON_PASSWORDS
    if COMMON_PASSWORDS == None:
        with open("path-to-file") as f:
            COMMON_PASSWORDS = f.readlines()
    return password in COMMON_PASSWORDS
```

How does that sound?

Reply:
Oh! Ok! Great!